### ISSUE TITLE
Work with omnidata normal/depth DPT networks weights?

### ISSUE THREAD

**Issue by pablovela5620** [YuiNsky/Multi-resolution-Monocular-Depth-Map-Fusion-by-Self-supervised-Gradient-based-Composition#2]
Excuse my ignorance, I would love to use the weights from [omnidata](https://github.com/EPFL-VILAB/omnidata/tree/main/omnidata_tools/torch), my understanding is that they use the same hybrid DPT network architecture to generate high resolution normal and depth maps. 

I see that they use pytorch lightning and that would require converting the ckpt models to .pt, but I guess my bigger question is would this also work for the normal estimation network? I don't fully understand how this model works so I wanted to ask. Thanks!

**Comment by YuiNsky** [comment]
My apologies for the delayed respons.
You can test your network with different input resolution and see if it shows the phenomenon like we mentioned in the paper. 
Many CNN estimation networks show such ability to predict richer details when you testing img is larger than the training design.

**Comment by zhanghongyong123456** [comment]
> My apologies for the delayed respons. You can test your network with different input resolution and see if it shows the phenomenon like we mentioned in the paper. Many CNN estimation networks show such ability to predict richer details when you testing img is larger than the training design.

The resulting video depth map is very flickering.  Is there any way to avoid flickering

**Comment by YuiNsky** [comment]
Our method is based on monocular estimation. Since the backbone was not trained to predict consecutive frames, it is difficult for us to avoid flickering.

**Comment by pablovela5620** [comment]
Thanks for the reply, do you know if this also works for normal estimation networks (my guess is not due to (H,W,3) v.s. depth map being (H,W,1)). Would to use this to upscale both depths and normals

**Comment by YuiNsky** [comment]
> Thanks for the reply, do you know if this also works for normal estimation networks (my guess is not due to (H,W,3) v.s. depth map being (H,W,1)). Would to use this to upscale both depths and normals

Sorry i did not test on normal, but i think the upscale operation seems like put estimation networks closer while expand its field of view. So there might be the same observation (better edges, worse value) when estimating normals.

**Comment by pablovela5620** [comment]
Great, thanks for the response
